apps/strapi-cms/package.json

```
@@ -0,0 +1,31 @@
+{
+  "name": "strapi-cms",
+  "private": true,
+  "version": "0.1.0",
+  "description": "A Strapi application",
+  "scripts": {
+    "dev": "strapi develop",
+    "develop": "strapi develop",
+    "start": "strapi start",
+    "build": "strapi build",
+    "strapi": "strapi"
+  },
+  "devDependencies": {},
+  "dependencies": {
+    "@strapi/strapi": "4.13.6",
+    "@strapi/plugin-users-permissions": "4.13.6",
+    "@strapi/plugin-i18n": "4.13.6",
+    "better-sqlite3": "8.6.0"
+  },
+  "author": {
+    "name": "A Strapi developer"
+  },
+  "strapi": {
+    "uuid": "1ae9c07f-4fbf-400a-9d32-bba36247aed0"
+  },
+  "engines": {
+    "node": ">=16.0.0 <=20.x.x",
```

```suggestion
    "node": ">=16.0.0",
```
I think we can remove the upper bound, because [Strapi is compatible also with version 20 of Node.JS](https://docs.strapi.io/dev-docs/installation/cli#preparing-the-installation)